api/generic-cti/answer-event.rst

```diff
@@ -0,0 +1,206 @@
+Call answered
+=============
+
+.. include:: /api/generic-cti/cti-endpoint-limitations.include.rst
+
+.. list-table:: Available attributes and sample data for answered events
+   :widths: 20, 40, 40
+   :header-rows: 1
+
+   * - Attribute
+     - Possible value
+     - Description
+   * - ``event``
+     - ``answer``
+     - Tell Zammad there's a new call
+   * - ``from``
+     - e.g. ``493055571600``, ``02214710334``, ``anonymous``
+     - Number that initiated the call
+   * - ``to``
+     - e.g. ``49221470334``, ``03023125771``
+     - Number that is being called
+   * - ``direction``
+     - ``in`` or ``out``
+     - The call direction - if your agent initiates a call this will be ``out``
+   * - ``callId``
+     - e.g. ``53ba82e2bd6d12d9fb2d3838f0cfb070``, ``5fb9532f40da834a``,
+       ``123456789``
+     - This is the CallID from the ``newCall`` event.
```

Comment:
- This is the identifier of the call session (which is also used in any other event which is related to a call session).

Reply:
I've taken the version from newCall because it technically describes it well enough as well in my opinion:

```
       An ID that is unique for the call. Zammad will use this ID to identify
       an existing call with following actions
       (e.g. like answering or hanging up)

       *This ID must be unique per call session*.
 ```